include/HUDL/HUDL.hpp

```diff
@@ -103,9 +103,15 @@ class HUDL {
 
     uint32_t thermTemps[4] = {};
 
-    static constexpr uint16_t OBJECT_DICTIONARY_SIZE = 30;
+    uint16_t statusWord = 0;
+    uint32_t positionActual = 0;
+    uint16_t torqueActual = 0;
+    uint32_t velocityValue = 0;
```

Comment:
All of these values, `totalVoltage` to `velocityValue`, should have a comment explaining what they are and where they come from.